app/recordtransfer/templates/recordtransfer/submission_form_review.html

```diff
@@ -38,7 +38,9 @@
                                                 {% if forloop.counter <= 5 %}
                                                     <div class="file-entry mb-2 flex items-center gap-2">
                                                         <i class="fas fa-file text-primary/60"></i>
-                                                        <a class="non-nav-link text-primary/60 font-medium underline hover:text-primary transition"
+                                                        <a class="non-nav-link text-primary/60 font-medium underline hover:text-primary transition break-all block"
+                                                           style="word-break: break-all;
+                                                                  max-width: 100%"
```

Comment:
You should be able to just use Tailwind classes rather than include inline styles.

1. For `word-break: break-all;`, the `break-all` class you've included already does that.

![Image](https://github.com/user-attachments/assets/447c2fe7-354d-42f5-bfb0-0481214ebba4)

2. For `max-width: 100%;`, you could just use [max-w-full](https://tailwindcss.com/docs/max-width#using-a-percentage)

```suggestion
                                                        <a class="non-nav-link text-primary/60 font-medium underline hover:text-primary transition break-all max-w-full block"
```

Reply:
That makes a lot more sense now — I was initially confused about why the DaisyUI classes didn’t seem to be working, but it turns out I was testing on the wrong server at the time. That was my oversight. This approach definitely simplifies the code and works as expected!

Resolved by 
 https://github.com/NationalCentreTruthReconciliation/Secure-Record-Transfer/pull/891/commits/6e4914ef70ffee0ed992770ae7630d4a3b43a9ea